src/surface/gui/gui/widgets/seagrass.py

```diff
@@ -13,6 +14,8 @@
 
 from gui.widgets.video_widget import CameraDescription, CameraType, PauseableVideoWidget
 
+N = 8
```

Comment:
```suggestion
GRID_DIMENSION = 8
```

Remember to update the other occurrences